infrasim/model/elements/drive.py

```diff
@@ -79,6 +80,15 @@ def precheck(self):
             raise ArgsNotCorrect("[CBaseDrive] sector_size {} is only supported 'raw' format"
                                  .format(self.__sector_size))
 
+        if os.path.exists(self._drive_info.get("file", "")) and self._drive_info.get("size"):
+            cmd = "qemu-img info {} --output json".format(self._drive_info.get("file"))
+            img_size = json.loads(run_command(cmd)[1])["virtual-size"]
+            if img_size != self._drive_info.get("size") * 1024 * 1024 * 1024:
```

Comment:
One situation we met is: define a drive with size of 1200.24GB which make the cacacity of 1288747886837.76 bytes; while the qemu-img output on image file is 1288747887104 bytes. The warning will be thrown here~. Should we work around this?

Reply:
ceil(1200.24 * 1024 * 1024 * 1024 / 512) * 512=1288747887104

Reply:
👍